lxd/instance.go

```diff
@@ -324,6 +325,7 @@ func instanceCreateAsCopy(s *state.State, opts instanceCreateAsCopyOpts, op *ope
 				// leave alone so we don't prevent copy.
 			}
 
+			srcSnap.LocalConfig()["volatile.uuid"] = uuid.New()
```

Comment:
The commit message is " instance: volatile.uuid remain unchanged for snapshot/backup restores " and yet this is changing the volatile.uuid, so it suggests it should be in a separate commit (if at all).